src/Evolution/Executables/ScalarAdvection/EvolveScalarAdvection.hpp

```diff
@@ -117,6 +117,10 @@ struct EvolutionMetavars {
                           Tags::AnalyticSolution<initial_data>,
                           Tags::AnalyticData<initial_data>>;
 
+  // tag for testing new initial data base class
+  using new_initial_data_tag =
```

Comment:
I think it's better to delete this alias and put the tag in the `const_global_cache_tags` list directly, so just drop this change from the commit.